Sources/MapboxMaps/Foundation/Extensions/Core/MapSnapshotOptions.swift

```diff
@@ -10,19 +10,21 @@ extension MapSnapshotOptions {
     ///         `.ideographsRasterizedLocally`
     ///   - resourceOptions: Resource fetching options to be used by the
     ///         snapshotter. Default uses the access token provided by
-    ///         `CredentialsManager.default`
+    ///         `ResourceOptionsManager.default`
     public convenience init(size: CGSize,
                             pixelRatio: CGFloat,
-                            glyphsRasterizationOptions: GlyphsRasterizationOptions? = GlyphsRasterizationOptions(fontFamilies: []),
-                            resourceOptions: ResourceOptions = ResourceOptions()) {
+                            glyphsRasterizationOptions: GlyphsRasterizationOptions? = GlyphsRasterizationOptions(),
+                            resourceOptions: ResourceOptions = ResourceOptionsManager.default.resourceOptions) {
```

Comment:
MapSnapshotOptions is a gl-native generated class in Android, and we couldn't override the default value here...